openml-python-common/src/main/java/com/feedzai/openml/python/jep/instance/JepInstance.java

```diff
@@ -97,8 +101,18 @@ public void stop() {
      */
     @Override
     public void run() {
-
-        try (final Jep jep = new Jep(false)) {
+        final Set<String> sharedModules = ImmutableSet.<String>builder()
+                .add("tensorflow")
+                .add("numpy")
```

Comment:
Please put these modules in a default XML file instead?

That way the user can remove them (e.g., what if he does not have tensorflow installed?)

Reply:
I prefer to include this modules hardcoded. The module doesn't have to be installed to be declared as a shared module (e.g. this test was executed in CI and the docker image used for that doesn't contain tensorflow installed).

Also, by having these modules hardcoded we wouldn't have to add the xml file to the library directories of the repositories that depend on this module.